src/IntegrationUtils.ts

```diff
@@ -257,3 +310,19 @@ export async function isCamelKAvailable(): Promise<boolean> {
 		resolve(false);
 	});
 }
+
+function validateDependency(text : string) {
+	let inputEmpty : boolean = (text === undefined || text.trim().length === 0);
+	if (inputEmpty) {
+		return 'Dependency must be specified in the form of an Apache Camel component id or a Maven dependency in the form of group:artifact:version';
+	}
+	if (!inputEmpty && text.indexOf(':') > -1) {
+		let trifold : string[] = text.split(':');
+		// make sure we have three colon-delimited strings
+		return !(trifold && trifold.length === 3) ? 'Dependency must be specified in the form of a Maven dependency as group:artifact:version' : null;
+	} else if (!(!inputEmpty && text.indexOf('-') > -1)) {
+		// assume we're a camel component for now
+		return 'Dependency must be specified in the form of an Apache Camel component id or a Maven dependency in the form of group:artifact:version';
```

Comment:
maybe the text is confusing?

Reply:
Recommendation? Maybe change to "Apache Camel component Artifact Id"?

Reply:
Updated